src/Stripe.net/Services/Checkout/SessionCreateOptions.cs

```diff
@@ -0,0 +1,86 @@
+namespace Stripe.Checkout
+{
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class SessionCreateOptions : BaseOptions
+    {
+        /// <summary>
+        /// Specify whether Checkout should collect the customer’s billing address. If set to
+        /// <c>required</c>, Checkout will always collect the customer’s billing address. If left
+        /// blank or set to <c>auto</c> Checkout will only collect the billing address when
+        /// necessary.
+        /// </summary>
+        [JsonProperty("billing_address_collection")]
+        public string BillingAddressCollection { get; set; }
+
+        /// <summary>
+        /// The URL the customer will be directed to if they decide to go back to your website.
+        /// </summary>
+        [JsonProperty("cancel_url")]
+        public string CancelUrl { get; set; }
+
+        /// <summary>
+        /// A unique string to reference the Checkout Session. This can be a customer ID, a cart
+        /// ID, or similar. It is included in the <c>checkout.session.completed</c> webhook and can
+        /// be used to fulfill the purchase.
+        /// </summary>
+        [JsonProperty("client_reference_id")]
+        public string ClientReferenceId { get; set; }
+
+        /// <summary>
+        /// ID of the customer this Checkout Session is for if one exists. May only be used with
+        /// <c>LineItems</c>. Usage with <c>SubscriptionData</c> is not yet available.
+        /// </summary>
+        [JsonProperty("customer")]
+        public string CustomerId { get; set; }
```

Comment:
I'm not sure if we're super consistent w/ regards to ordering the attributes alphabetically per attribute name (`CustomerId`) or per JSON name (`customer`), but in `Session`, `CustomerId` appears after `CustomerEmail`. Can you quickly check if there's a common style we use for this and update the PR to be consistent?

Reply:
There's no style beyond "remi does not know his alphabet". Will fix.

Reply:
Ha okay I know why. It's because if something is expandable then it ends up appearing first. But I'd rather re-alphabetize when it becomes expandable.